test/index.test.js

```diff
@@ -49,13 +51,9 @@ test('should add all resolved imports as dependencies', async () => {
     },
   });
 
-  expect(dependencies.sort()).toEqual(
-    [
-      lessFixturePath('import.less'),
-      lessFixturePath('css.css'),
-      lessFixturePath('basic.less'),
-    ].sort()
-  );
+  expect(dependencies).toContain(lessFixturePath('import.less'));
```

Comment:
webpack 5 adds more dependencies, so we need to check whether or not the loader has added them into the array, rather than the whole array equalling a fixed set of dependencies